tests/succeed/distillation/fresh-names.snap

```diff
@@ -0,0 +1,12 @@
+stdout = '''
+def a : () : Type = ();
+def const1 : fun (@A : Type) (@B : Type) -> A -> B -> A =
+fun @b @c x y => let x1 : b = x;
```

Comment:
I wonder if we could actually pull these binder names from the type somewhere?

Reply:
What do you mean?

Reply:
Like, try to use `A` and `B`. Just a thought though!